src/functions/theme.php

```diff
@@ -11,6 +11,9 @@
 function theme(string $hook, array $variables = []): MarkupInterface|string
 {
     $build['#theme'] = $hook;
+    if (function_exists('retrofit_preprocess_' . $hook)) {
+        ('retrofit_preprocess_' . $hook)($variables);
+    }
```

Comment:
We're decorating the theme registry, is there a way to just do this there? It's how we register legacy "process" hooks.

Reply:
This was moved, but I forgot to delete this.